### ISSUE TITLE
Inconsistent result between `uvloop.server.Server.sockets` and `asyncio.Server.sockets` in Python 3.5 and 3.6

### ISSUE THREAD

**Issue by mhchia** [MagicStack/uvloop#231]
* **uvloop version**: 0.11.2
* **Python version**: 3.6.8
* **Platform**: macOS Mojave 10.14.3
* **Can you reproduce the bug with `PYTHONASYNCIODEBUG` in env?**: Yes

According to the [Python document](https://docs.python.org/3.5/library/asyncio-eventloop.html#asyncio.Server.sockets) from 3.5~3.6, `asyncio.Server.sockets` should return `None` if the server is closed. But with uvloop as the event loop, `sockets` of a closed `Server` returns `[]`, which is the behavior since 3.7 from my understanding.

To reproduce:
```python
import asyncio

import uvloop


asyncio.set_event_loop_policy(uvloop.EventLoopPolicy())


async def close_server():
    def handler(reader, writer):
        pass
    s = await asyncio.start_server(handler, host='127.0.0.1', port=34567)
    s.close()
    await s.wait_closed()
    assert s.sockets is None


asyncio.get_event_loop().run_until_complete(close_server())
```

I found this code snippet fails in python 3.5~3.6. I'm not sure whether it is the desired behavior. If not, I'm also not sure how to fix it accordingly, since this behavior is consistent with 3.7+. I'm interested in the thoughts from you guys. Sorry if it is inappropriate and thanks a lot for reading.

**Comment by 1st1** [comment]
I think it's a rather minor inconsistency that shouldn't cause trouble, isn't it?  How did you discover this?

**Comment by mhchia** [comment]
Thank you for your reply!

> I think it's a rather minor inconsistency that shouldn't cause trouble, isn't it?

Agree. I made an [issue](https://bugs.python.org/issue36074) in CPython for the document of 3.7+, about the fact that they mentioned "List of socket.socket objects the server is listening on, or None if the server is closed.", which is not consistent with the implementation. The issue was fixed so the behavior of `uvloop.server.Server.sockets` is aligned with the document of python 3.7+. So there is no issue now:)

> How did you discover this?

I checked `asyncio.Server.sockets is None` to ensure the server is closed, in Python 3.6. It works, but fails right after I changed to use uvloop.
